internal/kokoro/continuous.sh

```diff
@@ -56,7 +56,8 @@ set -eo pipefail
 set -x
 
 # cd to project dir on Kokoro instance
-cd github/google-cloud-go
+# TODO(#6444): change this back to github/google-cloud-go before merging to main
```

Comment:
Need to remove this.

Reply:
Done in [9757864](https://github.com/googleapis/google-cloud-go/pull/6465/commits/9757864c9a0704e5f33257cefef95ce06de6f10f)